### ISSUE TITLE
Use dust limit for minimum coin value

### ISSUE THREAD

**Issue by cculianu** [KarolTrzeszczkowski/Inter-Wallet-Transfer-EC-plugin#8]
Hey man was looking over your plugin to see if I can add to it or plug extant bugs. This is because I plan on using it soon myself to transfer around some coins in a private manner.

https://github.com/KarolTrzeszczkowski/Inter-Wallet-Transfer-EC-plugin/blob/a2400b05d548248993878c55bb500d57e71cbe03/inter-wallet-transfer/ui.py#L184

You may want to just use dust limit for this. You can access it via either: 

```
wallet_instance.dust_threshold()

# *or* 

from electroncash.wallet import dust_threshold

#  ... 

dust_threshold(None)

```

Basically an output can't be smaller than that value (hard-coded to 546 satoshis for now).

**Comment by cculianu** [comment]
The reason for this is if the output ends up being 545 satoshis or smaller, the tx will fail on broadcast....

The check now makes sure it's >= 0.  You really want it to be >= `dust_threshold()`
